config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+
+
+  get '/', to: 'landing#index', as: 'landing'
```

Comment:
it provides a shortcut for accessing this path from anywhere in your code, not necessary now but could be useful later, you think I should remove it?